requirements.txt

```diff
@@ -3,8 +3,8 @@ numpy>=1.14.0,<2.0.0
 oauth2client<4.0.0
 pandas>=0.22.0,<1.0.0
 six>=1.11.0,<2.0.0
-tensorflow==1.13.1; python_version=='2.7'
-tensorflow-transform==0.12.0; python_version=='2.7'
-tensorflow-data-validation==0.12.0; python_version=='2.7'
+tensorflow==1.13.1
+tensorflow-transform==0.13.0
+tensorflow-data-validation==0.13.0
 tensorflow_metadata==0.12.1
```

Comment:
Let's bump this too? https://github.com/tensorflow/metadata/releases/tag/v0.13.0